### PR TITLE
add mui dropdown to navbar

### PR DESCRIPTION
**Pull request by chrispetito**
# Description

Add a navbar dropdown to replace the sign out button
Add 'my apps' and 'edit profile' to dropdown

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/Auth/AuthButton.js [hunk 1]
```diff
@@ -33,14 +45,48 @@ const AuthButton = ({ first_name }) => {
...
       {userIDCookie && (
         <>
-          {first_name && (
-            <label style={{ color: "#0c3c78", fontSize: "1.1rem" }}>
-              Hello {first_name}!
-            </label>
-          )}
-          <Button onClick={() => logoutWithRedirect()} color="secondary">
-            Sign out
+          {" "}
```
Comment (lappjeff) [inline src/components/Auth/AuthButton.js]: Are these curly bracket blocks doing anything?

File: src/components/Auth/AuthButton.js [hunk 2]
```diff
@@ -33,14 +45,48 @@ const AuthButton = ({ first_name }) => {
...
+          <Button
+            aria-controls="simple-menu"
+            aria-haspopup="true"
+            onClick={handleClick}
```
Comment (lappjeff) [inline src/components/Auth/AuthButton.js]: How is this working if we're not passing in an `e` argument?

File: src/components/Auth/AuthButton.js (hunk 1 above)
Comment (chrispetito) [reply src/components/Auth/AuthButton.js]: Removed!

File: src/components/Auth/AuthButton.js (hunk 2 above)
Comment (chrispetito) [reply src/components/Auth/AuthButton.js]: MUI docs show no argument is needed.

**Pull request merged**